### ISSUE TITLE
revisions to cell cycle signalling pathway term names

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#19985]
regulation of mitotic cell cycle phase transition (GO:1901990)
really refers largely  to a group of signalling pathways, although it does not mention signalling in the term name which can be confusing.

However, most of the descendants do:
i.e.
regulation of G2/M transition of mitotic cell cycle (GO:0010389)
Definition 
Any signalling pathway that modulates the activity of a cell cycle cyclin-dependent protein kinase to modulate the switch from G2 phase to M phase of the mitotic cell cycle.

One of the outstanding pieces of work from the cell cycle revamp a few years ago was to rename these so the they clearly represent signalling pathways.

The pattern should be:
regulation of G2/M transition of mitotic cell cycle (GO:0010389) ->
regulation of mitotic G2/M transition signalling

positive regulation of G2/M transition of mitotic cell cycle -> 
positive regulation of mitotic G2/M transition signalling

negative regulation of G2/M transition of mitotic cell cycle ->
negative regulation of mitotic G2/M transition signalling

regulation of G2/M transition of mitotic cell cycle involved in cellular response to nitrogen starvation
this particular one is a bit of a mouthful and is more commonly referred to as 
endosulphine signalling pathway so these  terms could be
regulation of endosulphine signalling pathway with the existing name *and*
regulation of  mitotic G2/M transition signalling  involved in cellular response to nitrogen starvation
as a synonym

Similarly all of the checkpoint signalling pathway terms names coudll be revised

signal transduction involved in mitotic G2 DNA damage checkpoint ->
mitotic G2 DNA damage checkpoint  signalling

(this will purge more of the 'involved in' clauses that make these appear to be compound terms, when they aren't really.

No hurry for this, we can do it on a call one day.

**Comment by ValWood** [comment]
others to fix
negative regulation of G2/MI transition of meiotic cell cycle
signal transduction involved in G2 cell size control checkpoint
signal transduction involved in mitotic DNA damage checkpoint

**Comment by mah11** [comment]
> The pattern should be:
> regulation of G2/M transition of mitotic cell cycle (GO:0010389) ->
> regulation of mitotic G2/M transition signalling

This seems to change the meaning such that the new names don't quite match the text definitions. At present the text defs say that signaling happens, and regulates a cell cycle phase transition, whereas the new names would say that signaling is the thing being regulated.

Which is the actual intended meaning?

If the current text defs are accurate, then the names would have to say something like

 "signaling pathway regulating mitotic G2/M transition" or
 "regulation of mitotic G2/M transition by signaling" or
 "mitotic G2/M transition regulation signaling"
 "mitotic G2/M transition regulatory signaling"

or maybe someone can come up with less clunky wording. But in any case the wording has to say it regulates the transition.

On the other hand, if the terms are meant to represent regulation of signaling pathways, then the text defs need to change.

**Comment by ValWood** [comment]
Yes that's not right. I will need to speak to Jacky about what is the "G2/M transition" and what is upstream. I think we may have it a bit wrong.
I have it on my todo list.

**Comment by pgaudet** [comment]
@ValWood do you want to ask Jackie about this one ? Would make sense to continue this work soon

**Comment by ValWood** [comment]
Yes I will speak to here soon, specifically about G1, but there are multiple things to address in this area, I might need to put some of this on hold for a few weeks. I will keep it near the top of my list though. Of course it might not be as complicated as I envisage...

**Comment by ValWood** [comment]
A comment from
https://github.com/geneontology/go-ontology/issues/12644#issuecomment-366282636

This is quite a big issue.

The cell cycle transitions are signalling pathways that converge on cdc2. IMO these terms should only be used to annotate the signalling pathways components.
see image in this ticket for G2/M, lots of signalling pathways converge on cdc2)
pombase/curation#1049

Unfortunately many researchers in higher organisms say any cell cycle block is regulating a cell cycle transition (or perhaps they don't say that, but curators interpret the cell cycle block in this way). These annotations include anything which, when mutated causes a cell cycle block. In fission yeast this would be upwards of 500 gene products if we made these annotations, and the cell cycle terms would cease to be useful to cell cycle researchers.

The genes which when mutated cause a cell cycle block are not really regulating the transitions at all. They are causing a problem which activates a checkpoint to block cell cycle progression. Only the checkpoint is involved in regulation of the transition in these cases. These experiments are demonstrating that there is a problem, and the checkpoint is activated.

Midori got closest to articulatin what has been in my head in this thread pombase/fypo#3172
whilst we were thinking about this for phenotype terms

So, ideally the G2/M transition term would be a child of
"signal transduction" it is a signalling pathway.

The problem is that many genes are annotated that cell cycle researchers would not consider to be regulating the G2/M transition. This would be difficult to deal with. So, I'm a bit stuck how to fix the problem.....
